core/trino-main/src/main/java/io/trino/execution/DropTableTask.java

```diff
@@ -20,16 +20,23 @@
 import io.trino.metadata.QualifiedObjectName;
 import io.trino.metadata.RedirectionAwareTableHandle;
 import io.trino.security.AccessControl;
+import io.trino.spi.TrinoException;
+import io.trino.spi.connector.CatalogSchemaTableName;
+import io.trino.spi.connector.DropRedirected;
+import io.trino.spi.connector.DropResult;
 import io.trino.sql.tree.DropTable;
 import io.trino.sql.tree.Expression;
 
 import javax.inject.Inject;
 
 import java.util.List;
+import java.util.Optional;
 
 import static com.google.common.util.concurrent.Futures.immediateVoidFuture;
 import static io.trino.metadata.MetadataUtil.createQualifiedObjectName;
+import static io.trino.metadata.QualifiedObjectName.convertFromSchemaTableName;
 import static io.trino.spi.StandardErrorCode.GENERIC_USER_ERROR;
+import static io.trino.spi.StandardErrorCode.METADATA_NOT_FOUND;
```

Comment:
Do note that you introduce this error code only in the subsequent commit https://github.com/trinodb/trino/pull/15065/commits/0eb6112b6d0d736fb1366fcf817e82d903872b0b

Reply:
`ci / check-commits` should catch it (fail)